www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
I think so, because the exact versions that users install are tested together. And everything is built against the exact numpy version that is used by the user. With ``pip`` it's a more random mix.

The more important reason is less moving parts though: just download one thing and it all works. Plus you get MKL with Anaconda, which also is nice.